packages/server-express/src/routes/routers.ts

```diff
@@ -54,9 +54,15 @@ routers.post('/challenge/:id', (req: Request, res: Response) => {
 routers.post('/cert/:id', (req: Request, res: Response) => {
   container.resolve<Controllers>(diControllers).getCertificate(req, res, req.params.id);
 });
+routers.get('/cert/:id', (req: Request, res: Response) => {
```

Comment:
shouldn't we have caCert instead, so it's clear from an API perspective that only CA cert could be obtained this way, WDYT?

Reply:
The reason of it is that when you are getting enrolled certificate in PKIX format, it adds links to response to the next certificates in chain. And current implementation uses the same controller for their getting. This is why I use the same controller with the same name

Reply:
we could use same controller, but different public API endpoint